core/src/main/scala/org/apache/spark/h2o/H2OContext.scala

```diff
@@ -329,6 +329,17 @@ class H2OContext private(val sparkSession: SparkSession, conf: H2OConf) extends
     basic ++ sparkYarnAppId ++ backend.epilog
   }
 
+  /**
+    * @param path directory where the logs will be downloaded
+    */
+  def downloadH2OLogs(path: String, logFileName: String = "logs.zip"): String = {
```

Comment:
`downloadH2OLogs` instead of `downloadAllLogs` is used on purpose as in the case of `downloadAllLogs` the user might be confused that it would download also Spark logs